src/OmniSharp.Cake/OmniSharp.Cake.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+    <WarningsAsErrors>true</WarningsAsErrors>
```

Comment:
This isn't needed (and it's actually not the right property name). It will already be set automatically by https://github.com/OmniSharp/omnisharp-roslyn/blob/master/build/Settings.props#L5.